shared/js/initApp.js

```diff
@@ -172,21 +172,64 @@ dashboard.saveProject = function(callback) {
       location.hash = dashboard.currentApp.id = data.id;
       callbackSafe(callback, data);
     });
-    dashboard.showProjectHeader();
+    if (!dashboard.currentApp.hidden) {
```

Comment:
Actually this comment should be moved down to [line 230](https://github.com/code-dot-org/code-dot-org/pull/1415/files#diff-266be2044ca11fab972603fe85be50b5R230) below.  This occurrence can be removed since it should already be visible from the call in `initApp()`.